src/shared/containers/VideoPluginContainer/VideoPluginContainer.tsx

```diff
@@ -59,27 +59,33 @@ const VideoPluginContainer: React.FunctionComponent<VideoProps> = ({
     setIsModalVisible(true);
   };
   const loadVideo = async () => {
-    const videoResource = (await nexus.Resource.get(
-      orgLabel,
-      projectLabel,
-      encodeURIComponent(resource['@id'])
-    )) as Resource<{
-      video: VideoObject[];
-    }>;
+    try {
+      const videoResource = (await nexus.Resource.get(
+        orgLabel,
+        projectLabel,
+        encodeURIComponent(resource['@id'])
+      )) as Resource<{
+        video: VideoObject[];
+      }>;
 
-    const videoData = videoResource.video
-      ? ([videoResource.video].flat() as VideoObject[])
-      : undefined;
+      const videoData = videoResource.video
+        ? ([videoResource.video].flat() as VideoObject[])
+        : undefined;
 
-    setVideoData(videoData);
+      setVideoData(videoData);
+    } catch (e) {
+      notification.info({
+        message: 'Error fetching video data',
+      });
+    }
   };
 
   return (
     <Collapse
       activeKey={collapsed ? 'video' : undefined}
       onChange={e => handleCollapseChanged()}
     >
-      <Panel header="Video" key="video">
+      <Panel header="Video" key="video" forceRender={true}>
```

Comment:
Avoids lazy loading. This mostly to help with testing. Won't affect the behavior in any significant way and will not make any extra network call.